Gemfile

```diff
@@ -46,3 +46,4 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+gem 'devise'
```

Comment:
Add a gem version.

Also, configure your text editor to add a new line at the end of each file.